src/engine/srv.c

```diff
@@ -1248,8 +1248,8 @@ static int
 dss_sys_db_init()
 {
 	int	 rc;
-	/* walkaround*/
-	char	*lmm_db_path = getenv("DAOS_LMM_DB_PATH");
+	/* NVMe config should always reside in an engine specific writable location */
+	char	*lmm_db_path = dirname(dss_nvme_conf);
```

Comment:
This requires control plane pass in the path of dss_nvme_conf located in external storage, right?

Reply:
Yes and this is already done